src/components/SearchMap.js

```diff
@@ -1,20 +1,26 @@
 // 3rd party libraries - core
 import React from 'react';
-import {View, Image, StyleSheet, ActivityIndicator} from 'react-native';
-import {Icon} from 'react-native-elements';
+import {View, StyleSheet, ActivityIndicator, Dimensions, ImageBackground, TouchableOpacity} from 'react-native';
+import {Icon, Card, Text} from 'react-native-elements';
 import {MapView} from 'expo';
 
 const {Marker} = MapView;
-// 3rd party libraries - additional
 import _ from 'lodash';
 
+// 3rd party libraries - additional
+
 // styles and language
-import {campsiteIcon} from '../styles';
+import {campsite} from '../locale.en';
+
+const {campsite_form: {accessibility_options}} = campsite;
+
+import {featureIconDetails, facilityIconDetails} from "../constants";
 
-// our components - core
-// our components - additional
+import {campsiteIcon, selectedCampsiteIcon, navyBlueButton} from '../styles';
 
-const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites, navigate}) => {
+const SCREEN_WIDTH = Dimensions.get('window').width;
```

Comment:
@amgoodman5 heads up, not sure if you've worked with the `Dimensions` library, but it's a handy way to get the width of the screen if you need it.